package.json

```diff
@@ -10,11 +10,15 @@
   "scripts": {
     "prebootstrap": "node prebuild.js",
     "test": "lerna run test",
-    "bootstrap": "lerna bootstrap",
-    "lint": "lerna run lint",
-    "lint:js": "lerna run lint",
+    "bootstrap": "lerna bootstrap --npm-client=yarn --force-local",
     "build": "lerna run build",
     "deploy": "lerna run deploy --scope=vagov-documentation",
+    "lint": "npx vagov-eslint lint:js",
```

Comment:
Using `npm` here errors out. I'm not sure what `npm vagov-eslint` would target- my understanding is that `npx` runs scripts in `node_modules/bin` and that's there's not a way to run a `node_module`'s `package.json` scripts with an `npm` command.